src/mgr/OSDPerfMetricReport.h

```diff
@@ -0,0 +1,43 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+#ifndef OSD_PERF_METRIC_REPORT_H_
+#define OSD_PERF_METRIC_REPORT_H_
+#include "include/denc.h"
+
+#include "common/perf_counters.h"
+
+//#include "mgr/OSDPerfMetricQuery"
+
+typedef int OSDPerfMetricQueryID;  //Temporary; to be moved in mgr/OSDPerfMetricQuery.h ?
+
+struct PerformanceCounter {
+  std::string name;
+  perfcounter_type_d type;
+};
+
+
+struct OSDPerfMetricReport {
+  OSDPerfMetricQueryID query_id; 
```

Comment:
Mykola can add his query key (descriptor) here in his commit:
```suggestion
```

Reply:
Ok, removed!